model_hub/examples/mmdetection/hydra/configs/data/gcs.yaml

```diff
@@ -0,0 +1,3 @@
+file_client_args:
+  backend: gcs
+  bucket_name: ???
```

Comment:
should we host data for users as part of mmdetection?

Reply:
the datasets have licenses that make it difficult for us to host data